plugins/parent-expanded/index.js

```diff
@@ -0,0 +1,17 @@
+const selectorParser = require('postcss-selector-parser')
+
+module.exports = () => ({ addVariant, e }) => {
+  addVariant('parent-expanded', ({ modifySelectors, separator }) => {
+    return modifySelectors(({ selector }) => {
+      return selectorParser(selectors => {
+        selectors.walkClasses(sel => {
+          sel.value = `parent-expanded${separator}${sel.value}`
+          sel.parent.insertBefore(
+            sel,
+            selectorParser().astSync('[aria-expanded="true"] ')
+          )
+        })
```

Comment:
no change needed, just wondering if this works differently than doing this?
```
return `[aria-expanded="true"] .${e(
  `parent-expanded${separator}${selector}`,
)}`
```

Reply:
Not sure. I had pulled the `selectorParser` and `walkClasses` stuff from some core TW code a while back.